pom.xml

```diff
@@ -49,6 +49,7 @@ THE SOFTWARE.
   </licenses>
 
   <modules>
+    <module>bom</module>
```

Comment:
Did you forget to `git add bom/pom.xml`?

Reply:
oops